bedrock/base/templates/includes/sticky-promo.html

```diff
@@ -0,0 +1,38 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+<aside class="mzp-c-sticky-promo mzp-t-product-firefox mzp-t-dark">
+  <button class="mzp-c-sticky-promo-close" type="button">{{ ftl('ui-close') }}</button>
+  <div class="promo-firefox">
+    <h3 class="mzp-c-sticky-promo-title">{{ ftl('firefox-sticky-promo-get-the-latest-firefox') }}</h3>
+    {{ download_firefox_thanks(dom_id='sticky-promo', download_location='other', button_class='mzp-c-button mzp-t-product mzp-t-small') }}
+  </div>
+  <div class="promo-products">
+    <h3 class="mzp-c-sticky-promo-title">{{ ftl('firefox-sticky-promo-meet-our-family-of') }}</h3>
+    <ul class="promo-products-list">
+      <li>
+        <a data-link-name="Browsers" data-link-type="link" class="promo-products-link" href="{{ url('firefox.browsers.index') }}">
```

Comment:
For internal links – are there more tags needed? I saw other `data-link` attrs like: `data-link-position` and `data-link-group`.

Reply:
Maybe `data-link-position="sticky promo"`?